cmd/builder.go

```diff
@@ -49,13 +49,14 @@ var (
 
 // NewCmdVersion provides a shim around version for
 // non-client packages that require version information
-func NewCmdVersion(fullName string, versionInfo k8sversion.Info, out io.Writer) *cobra.Command {
+func NewCmdVersion(fullName string, versionInfo k8sversion.Info, buildahVersion string, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "version",
 		Short: "Display version",
 		Long:  "Display version",
 		Run: func(cmd *cobra.Command, args []string) {
 			fmt.Fprintf(out, "%s %v\n", fullName, versionInfo)
+			fmt.Fprintf(out, "Buildah version %s\n", buildahVersion)
```

Comment:
Print the buildah version when the `<cmd> version` command is invoked. Similar to what we do with `oc version`